code/modules/fissionreactor/fuelmaker.dm

```diff
@@ -0,0 +1,393 @@
+/*
+in this file:
+the machine which makes fuel rods have things in them.
+*/
+
+//because radon is a gas, we need to interface with gasses. yeah, this kind of sucks, but what are you gonna do? (inb4 make better code lol)
+/obj/machinery/atmospherics/unary/fissionfuelmaker
+	name="isotopic separational combiner" //just about the most technobable you could get.
+	var/datum/reagents/held_elements=new /datum/reagents
+	use_power = MACHINE_POWER_USE_IDLE
+	idle_power_usage = 200
+	anchored=1
+	density=1
+	active_power_usage = 1000
+	icon='icons/obj/fissionreactor/fuelmaker.dmi'
+	icon_state="fuelmaker"
+	var/hatchopen=FALSE
+	var/obj/item/weapon/fuelrod/heldrod = null
+	var/obj/item/weapon/reagent_containers/container=null
+	var/datum/html_interface/interface
+	var/pipename="isotopic separational combiner"
+	
+/obj/machinery/atmospherics/unary/fissionfuelmaker/proc/get_pipe_dir() //the atmos gods demand a sacrifice.
+	return dir	
+
+/obj/machinery/atmospherics/unary/fissionfuelmaker/New()
+	..()
+	src.buildFrom(usr,src)
+	interface=new /datum/html_interface(src,"isotopic separational combiner",550,500,"<link rel='stylesheet' href='fission.css'>")	
+	buildui()
+
+/obj/machinery/atmospherics/unary/fissionfuelmaker/attackby(var/obj/item/I,var/mob/user)
+	if(istype(I,/obj/item/weapon/fuelrod))
+		if(heldrod)
+			to_chat(user,"There's already a fuel rod inserted into \the [src].")
+		else
+			if(!user.drop_item(I))
+				return
+			to_chat(user,"You insert the fuel rod into \the [src].")
+			I.loc=null
```

Comment:
cannot be stored in the machine because the reactor houses it, and the destruction of reactors would feel really weird if destroying the controller cut all power

Reply:
Then pick a reactor housing to store it in when you place it in the controller

Reply:
oh wait, wrong machine, my bad